From b6ca3876e233c724fd460c1579abc4ab63c8d01e Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Wed, 4 Oct 2017 15:23:08 +0300
Subject: [PATCH] Disable valadoc

Valadoc is a documentation generator for Vala sources, which was
recently merged into the main vala source tree. Unsurprisingly,
it's broken in cross-compile environment in multiple ways,
so let's fix it some other time.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 Makefile.am | 2 --
 1 file changed, 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e5dc0cc..813b3fc 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -13,8 +13,6 @@ SUBDIRS = \
 	doc \
 	gobject-introspection \
 	vapigen \
-	libvaladoc \
-	valadoc \
 	$(NULL)
 
 if ENABLE_UNVERSIONED
-- 
2.14.1

